src/components/Card/LessonCard.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react'
+import PlayButton from '../Button/PlayButton'
+import Card from './'
+
+export const LessonMeta = ({meta}) => {
+  return (
+    <div className='flex items-center gray'>
+      <img src={meta.langImg} className='w2' alt='' />
```

Comment:
Can the `alt` have something useful for accessibility / bad network, like `{meta.{something}` instead of an empty string?